torch/_inductor/utils.py

```diff
@@ -902,11 +902,17 @@ def blue_text(msg):
     return _color_text(msg, "blue")
 
 
-PYTHON_TYPE_TO_SCHEMA_TYPE = {
-    torch.dtype: "int",
-    torch.device: "Device",
-    bool: "bool",
-}
+def python_type_to_schema_type():
+    from . import ir
```

Comment:
Converting dict to a function here because we need to use this local import `from. import ir` to avoid circular import.